### DISCUSSION TITLE
Add the Modrinth App to Flathub

### DISCUSSION THREAD

**Discussion by CaptainDeSync** [modrinth/.github#175] [Feature requests]
It would be amazing if the modrinth app was added to flathub so we can download the app way easier on steam decks. As of right now, the only solution is a ton of console work-arounds and its not very easy to say the least. The app will load in Desktop Mode, but in Game mode, it simply does not work.

**Comment by triphora** [comment]
Duplicate of modrinth/theseus#497

**Reply by CaptainDeSync** [reply]
Oops, sorry. Wasn't able to find that post so I made this.
